docs/source/en/quicktour.mdx

```diff
@@ -14,53 +14,22 @@ specific language governing permissions and limitations under the License.
 
 [[open-in-colab]]
 
-Get up and running with 🤗 Transformers! Start using the [`pipeline`] for rapid inference, and quickly load a pretrained model and tokenizer with an [AutoClass](./model_doc/auto) to solve your text, vision or audio task.
+The 🤗 Transformers library is built around four major concepts to maximize user-friendliness and flexibility:
 
-<Tip>
-
-All code examples presented in the documentation have a toggle on the top left for PyTorch and TensorFlow. If
-not, the code is expected to work for both backends without any change.
-
-</Tip>
-
-## Pipeline
-
-[`pipeline`] is the easiest way to use a pretrained model for a given task.
-
-<Youtube id="tiZFewofSLM"/>
+* Start using the [`pipeline`] for rapid inference.
+* Load a pretrained model from the [Hub](https://huggingface.co/models).
+* Customize a base configuration, preprocessor, and model to modify how a model is built.
+* Use the [`Trainer`] class - a PyTorch training loop optimized for 🤗 Transformers models - to train a model.
```

Comment:
The quicktour should present the same features for PyTorch and TensorFlow, TensorFlow is not a second-class citizen ;-) So this section should have a part for fine-tuning with the Trainer in PyTorch and model.fit in Keras.